doc/Language/classtut.pod6

```diff
@@ -5,86 +5,52 @@
 =SUBTITLE A tutorial about creating and using classes in Raku
 
 X<|Tutorial,OOP>
+Raku provides a rich built-in syntax for defining and using classes. It makes writing classes
+expressive and short for most cases, but also provides mechanisms to cover
+the rare corner cases.
 
-Raku has a rich built-in syntax for defining and using classes.
+=head1 A quick overview
 
-A default constructor allows the setting of attributes for the created
-object:
+Let's start with an example to give an overview:
 
 =begin code
-class Point {
-    has Int $.x;
-    has Int $.y;
-}
-
 class Rectangle {
-    has Point $.lower;
-    has Point $.upper;
+    has Int $.length = 1;
+    has Int $.width = 1;
 
     method area(--> Int) {
-        ($!upper.x - $!lower.x) * ( $!upper.y - $!lower.y);
+        return $!length * $!width;
     }
 }
 
-# Create a new Rectangle from two Points
-my $r = Rectangle.new(lower => Point.new(x => 0, y => 0),
-                      upper => Point.new(x => 10, y => 10));
-
-say $r.area(); # OUTPUT: «100␤»
+my $r1 = Rectangle.new(length => 2, width => 3);
+say $r1.area(); # OUTPUT: «6␤»
 =end code
 
-In the two classes, the default constructor is being used. This
-constructor will use named parameters in its invocation: C«Point.new(x =>
-0, y => 0)».
+We define a new I<Rectangle> class using the L<class|/language/objects#Classes> keyword. It has two L<attributes|Attributes>, C<$!length> and C<$!width> introduced with the L<has> keyword. Both default to C<1>. Read only accessor methods are automatically generated. (Note the C<.> instead of C<!> in the declaration, which triggers the generation. Mnemonic: C<!> resembles a closed door, C<.> an open one.)
```

Comment:
I don't think this mnemonic was intended (https://raku-advent.blog/2020/08/16/rfc-188-by-damian-conway-objects-private-keys-and-methods/) - I'd remote it.

Reply:
s/remote/remove/?

Irrespective of where I picked the mnemonic up (I connect it to Larry somehow. Maybe a talk?), or whether it was intended or not - I believe it's a fitting one. Why should we then not keep it?

I'm not deeply committed to it, so if you insist I can remove it.